src/types/account-members.d.ts

```diff
@@ -14,6 +14,10 @@ export interface AccountMemberBase extends Identifiable {
     email?: string
 }
 
+export interface AccountMemberCustom extends AccountMemberBase {
+    [key: string]: any
```

Comment:
This means that this interface can have any property of any type. Do we know the fields for this?